### ISSUE TITLE
Make sidebar donate link optional?

### ISSUE THREAD

**Issue by vitorio** [HelloZeroNet/ZeroNet#1467]
### Step 1: Please describe your environment

  * ZeroNet version: 0.6.3
  * Operating system: OS X 10.11, Windows 10
  * Web browser: Safari, Chrome, Edge
  * Tor status: not available on Mac/available on Windows
  * Opened port: yes

### Step 2: Describe the problem:

As a site owner, I'd like to be able to remove the "Donate" link in the sidebar for my site.

There doesn't appear to be a way to do this through the ZeroFrame API, perhaps it could be added as an optional flag in content.json?  Something like:

```json
"donate": false
```

or

```json
"hide_donate_links": true
```

?

**Comment by anoadragon453** [comment]
The ability to remove it and/or change it would be nice.

**Comment by Thunder33345** [comment]
well the bitcoin address is the site address, the site private key
the same key you used to access/edit the site
thus you already have a bitcoin address believe it or not

**Comment by anoadragon453** [comment]
Yes, but some people prefer to use their main btc address which may 
already be verified by a WoT.

**Comment by Thunder33345** [comment]
the site's address is always site owner, thus anyone can reasonably expect that if i donate to `1HeLLo4uzjaLetFx6NH3PMwFP3qbRbTf3D` the owner of that site can be able to withdraw it

and there's nothing stopping you from donating to any site address
https://www.blockchain.com/en/btc/address/1HeLLo4uzjaLetFx6NH3PMwFP3qbRbTf3D

**Comment by krixano** [comment]
Right, but some people don't want to have to import every current and new zite. They just want to use one for everything.

**Comment by vitorio** [comment]
So extending my original proposal, then, perhaps:

By default, the donate link exists, and it shows the site address against a Bitcoin URL handler.

If `content.json` contains a key of: `"donate": false` or `"donate": ""`, it does not display the site address or link.

If `content.json` contains a key of: `"donate": "1..."`, where that value passes a Bitcoin address validation, it treats it as a Bitcoin address and shows it against a Bitcoin URL handler.

If `content.json` contains a key of `"donate": "http://..."` or `"donate": "https://..."`, where those values are strings beginning with "http://" or "https://", it treats it as a URL and shows only the "Donate" button linking to that.

If `content.json` contains a string with any other contents, it displays it as a block of text, suitable for site owners to provide alternative donation instructions.

Thoughts?

**Comment by Thunder33345** [comment]
actually that's not bad proposal, adding customizability is a +

BUT what if someone want to use another cryptocurrency
say Ethereum, Bitcoin Cash. Doge or so on?
i think begin amongous and somehow support all crypto will be nice

why not even allow text like "Donate to ETH(insert address) BCH(insert address)"
when click it will show that as popup

**Comment by vitorio** [comment]
To be clear about my use case for this: in the US, having evidence of a profit motive can be the difference between a civil charge and criminal charge under the DMCA.  Being able to remove the "Donate" link and signal that you're not distributing content for financial gain may make it safer to do so.
